src/sonic_ax_impl/mibs/ietf/rfc2737.py

```diff
@@ -675,7 +675,7 @@ def _update_per_namespace_data(self, pubsub):
 
             db_entry = msg["channel"].split(":")[-1]
             data = msg['data']  # event data
-            if not isinstance(data, bytes):
+            if not isinstance(data, str):
```

Comment:
>str [](start = 36, length = 3)

This seems a valid fix. Wondering how it worked before? I think it hit a type runtime error below `"set" in data` ?

Reply:
it was b"set" before.